tests/pytorch/fused_attn/test_fused_attn_with_cp.py

```diff
@@ -43,9 +43,21 @@ def test_cp_with_flash_attention(dtype, model, qkv_format):
         check=True
     )
 
+model_configs_fused_attn = {
+    #   test:             b,  h, hg,   d,   sq,  skv,   p,      mask,              bias
+    "cp_1_0": ModelConfig(1, 12, 12, 128, 4096, 4096, 0.0,  "causal",         "no_bias"), # MHA
+    "cp_1_1": ModelConfig(1, 12, 12, 128, 4096, 4096, 0.0, "no_mask",         "no_bias"), # MHA
+    "cp_1_2": ModelConfig(1, 12, 12, 128, 4096, 4096, 0.0,  "causal", "post_scale_bias"), # MHA
+    "cp_1_3": ModelConfig(1, 12, 12, 128, 4096, 4096, 0.0, "no_mask", "post_scale_bias"), # MHA
+    "cp_2_0": ModelConfig(1, 12,  1, 128, 4096, 4096, 0.0,  "causal",         "no_bias"), # GQA
+    "cp_2_1": ModelConfig(1, 12,  1, 128, 4096, 4096, 0.0, "no_mask",         "no_bias"), # GQA
+    "cp_2_2": ModelConfig(1, 12,  1, 128, 4096, 4096, 0.0,  "causal", "post_scale_bias"), # GQA
+    "cp_2_3": ModelConfig(1, 12,  1, 128, 4096, 4096, 0.0, "no_mask", "post_scale_bias"), # GQA
+}
+
```

Comment:
I remember there was some testing issue with the h12_s16384 case. Has that been resolved?

Reply:
no, not yet. as I told you before, nobody asked for this specially, I just reported the issue to you while I found it.
I was OOTO, so did not get time to look into it.

Reply:
Can you file a bug with cuDNN please?

Reply:
Looked into our previous conversation: the issue wasn't to do with cuDNN but PyTorch's bmm() call. We can look into that separately from this PR.